Check for changes to authorships

awead opened the pull request:
Any changes to a creator, either its values or order, should count as a change to the version and will allow publishing.

Fixes #721

mtribone commented:
New work view

![screen shot 2017-07-13 at 10 30 24 am](https://user-images.githubusercontent.com/4163828/28171429-66d5efb8-67b6-11e7-9c0f-e7fb01bb8f40.png)

awead committed:
Check for changes to authorships

Any changes to a creator, either its values or order, should count as a
change to the version and will allow publishing.
